src/doc/rustc/src/platform-support.md

```diff
@@ -431,7 +431,7 @@ target | std | host | notes
 [`xtensa-esp32s2-none-elf`](platform-support/xtensa.md) | * |  | Xtensa ESP32-S2
 [`xtensa-esp32s3-espidf`](platform-support/esp-idf.md) | ✓ |  | Xtensa ESP32-S3
 [`xtensa-esp32s3-none-elf`](platform-support/xtensa.md) | * |  | Xtensa ESP32-S3
-[`i386-lynx-lynxos178`](platform-support/i386-lynx-lynxos178.md) | * |  | 32-bit LynxOS-178 (2022.07)
+[`i386-lynx-lynxos178`](platform-support/i386-lynx-lynxos178.md) | * |  | 32-bit x86 LynxOS-178 (2022.07)
```

Comment:
This line is not present in upstream at all, seems it should be removed?

Reply:
it's our custom target (and I moved it here so it stands out, and to avoid future conflicts)